io/cmd.go

```diff
@@ -83,13 +83,23 @@ func RunCmdWithOutputParser(config CmdConfig, prompt bool, regExpStruct ...*CmdO
 	if err != nil {
 		return
 	}
-	defer cmdReader.Close()
+	defer func() {
```

Comment:
The changes added to this file are already being done by @yahavi in a different PR.

Reply:
I didn't see it merged yet and it fails the CI , should i drop this commit?